tools/debian-live

```
@@ -641,20 +785,32 @@ function postinstall {
 	    relase="$release" \
 	    bash -c '
 		PS4="POSTINSTALL: "
+		export LC_ALL=C
+
 		set -xeuo pipefail
 
 		echo debian-live-$release >/etc/hostname
 		echo "TRITON_RELEASE=\"$release\"" >> /etc/os-release
 
+		apt-get install -y systemd-resolved
+		sed -i -e "s/^#DNSSEC=.*/DNSSEC=false/" /etc/systemd/resolved.conf
+
+		# This service will attempt to parse the ip parameter on the kernel
+		# command line and fail, it is not needed for Triton
+		rm -f /usr/lib/systemd/system/systemd-network-generator.service
```

Can we get by without this?

---

I'm open to other options if  you have any, I've tried quite a few things this was the only working solution I found.

---

I mean systemd-resolved itself. Can it operate without that package?

---

At the moment the PI cannot operate without it.

I tested removing it (to remind myself) and I can build a PI without it but the CN will be unable to resolve triton service names, the first symptom is that `triton-vm-agent.service` fails to start, unable to resolve vmapi.

Also, if I attempt to install it with along with the other `$packages` in the PI, the build will fail early as the chroot becomes unable to resolve DNS names and install packages which is why I added the install step in the postinstall step.